include/chimera/util.h

```diff
@@ -315,6 +315,8 @@ std::pair<unsigned, unsigned> getFunctionArgumentRange(
  */
 bool hasNonPublicParam(const clang::CXXMethodDecl *decl);
 
+std::string getPythonBuiltinTypeSpelling(const std::string &type_name);
```

Comment:
We should make this match whatever function name we end up using for the operator overloading.  It might have been `Name` instead of `Spelling`.

Reply:
Sounds good to me. Let me rename them both with `Name`.